src/panels/lovelace/common/compute-tooltip.ts

```diff
@@ -48,20 +56,31 @@ function computeActionTooltip(
     return "";
   }
 
-  let tooltip = isHold ? "Hold: " : "Tap: ";
+  let tooltip =
+    (isHold
+      ? hass.localize("ui.panel.lovelace.cards.picture-elements.hold")
+      : hass.localize("ui.panel.lovelace.cards.picture-elements.tap")) + " ";
 
   switch (config.action) {
     case "navigate":
-      tooltip += `Navigate to ${config.navigation_path}`;
+      tooltip += `${hass.localize(
+        "ui.panel.lovelace.cards.picture-elements.navigate_to"
+      )} ${config.navigation_path}`;
```

Comment:
Don't concatenate strings. Instead the localization string should have a placeholder.